alicloud/resource_alicloud_havip_attachment_test.go

```diff
@@ -0,0 +1,145 @@
+package alicloud
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAlicloudHaVipAttachment_basic(t *testing.T) {
```

Comment:
Please add a sweeper test and set it to havip and instance sweeper's dependence.